README.md

```diff
@@ -22,6 +22,12 @@ glance-docker-container-ext:
     - PORT=8081 # Optional, default is 8081
   volumes:
     - /var/run/docker.sock:/var/run/docker.sock
+# labels:
+#   glance.0.enable: true
+#   glance.0.name: Glance
+#   glance.0.description: Self-hosted dashboard that puts all your feeds in one place
+#   glance.0.url: https://glance.example.com/
+#   glance.0.icon: https://avatars.githubusercontent.com/u/159397742?s=48&v=4
```

Comment:
I think this isn't the right place to put this example
- this container is not supposed to have a widget in the glance homepage
- these labels are not required in this step, but in the configuration below